.gitignore

```diff
@@ -145,6 +145,7 @@ cython_debug/
 # Collected static files and media files
 hackathon_site/static/
 hackathon_site/media/
+hackathon_site/emails/
```

Comment:
we don't want to upload our email template?

Reply:
This directory is generated by Django's email backend for development purposes https://docs.djangoproject.com/en/4.1/topics/email/#topic-email-file-backend so we wouldn't want this in the repo bc the contents would be different for everyone. Similar to how uploading resumes work.